stdlib/Profile/src/Allocs.jl

```diff
@@ -98,7 +123,14 @@ objects which can be analyzed.
 function fetch()
     raw_results = ccall(:jl_fetch_alloc_profile, RawAllocResults, ())
     decoded_results = decode(raw_results)
-    @warn("This allocation profiler currently misses some allocations. " *
+
+    println("total: $(g_total_allocs[])")
+    println("estimated total: $(g_estimated_total_allocs[])")
+
+    missed_allocs = g_total_allocs[] - g_estimated_total_allocs[]
+    missed_percentage = round(Int, Float64(missed_allocs) / Float64(g_total_allocs[]) * 100)
+
+    @warn("This allocation profiler missed $(missed_percentage)% of allocs in the last run. " *
     "For more info see https://github.com/JuliaLang/julia/issues/43688")
```

Comment:
You should roll all of this into a single warning message for users.

Maybe something like this?
```suggestion
    missed_allocs = length(raw_results.allocs) - g_expected_sampled_allocs[]
    missed_percentage = round(Int, Float64(missed_allocs) / Float64(g_expected_sampled_allocs[]) * 100)

    @warn("The allocation profiler is not fully implemented, and missed $(missed_percentage)%" *
                 "($(missed_allocs) / $(g_expected_sampled_allocs[])) of allocs in the last run. " *
                 "For more info see https://github.com/JuliaLang/julia/issues/43688")
```

Reply:
(This is based on my suggestions from my previous comment, but you could do something similar with what you had too if you don't like my other suggesiton)

Reply:
yep, I like it. done